README.md

```diff
@@ -122,7 +122,13 @@ This is the list of available configuration options:
 | `POSTMARK_API_KEY` | API Key for sending emails when using Postmark |
 | `MAILGUN_API_KEY` | API Key for sending emails when using Mailgun |
 | `MAILGUN_DOMAIN` | Domain name from which emails are sent when using Mailgun |
+| `SMTP_HOST` | SMTP host from which emails are sent when using SMTP |
+| `SMTP_PORT` | SMTP port from which emails are sent when using SMTP |
+| `SMTP_SECURE` | SMTP TLS from which emails are sent when using SMTP ("true"/"false") |
+| `SMTP_USER` | SMTP user from which emails are sent when using SMTP |
+| `SMTP_PASS` | SMTP password from which emails are sent when using SMTP |
 | `TEMPLATES_DIR` | Absolute path to directory where templates will be found |
+| `TOKEN` | Secret Token to communicate the microservice and your app (optional) |
```

Comment:
if the token is optional, it should exist an option to disable it

Reply:
I think that if you don't send this environment variable, it is the optionality. Are you proposing a disabled value like "none" ?

That's a good idea.

Reply:
I think if you don't set the env var, `token !== env('TOKEN')` always returns 404 when you send a token. And, if you don't send a token and you has setted a `TOKEN` env, this `token !== env('TOKEN')` is not evaluated and the comprobation can be omitted (jumped).

So, yeah, I think a `USE_TOKEN` or something like that should be setted! :smile: